prow/plugins/trigger/trigger.go

```diff
@@ -254,6 +254,15 @@ func TrustedUser(ghc trustedUserClient, onlyOrgMembers bool, trustedOrg, user, o
 		}
 	}
 
+	// Determine if user is on trusted_apps list.
+	// This allows automatic tests execution for GitHub automations that cannot be added as collaborators.
+	for _, trustedApp := range trustedApps {
+		// username must explicitly end with either [bot] or [app]
+		if strings.HasSuffix(user, "[bot]") || strings.HasSuffix(user, "[app]") && user == trustedApp {
```

Comment:
`[app]` can also be used? I've only seen `[bot]` so far and our code only handles that. Do you have an example for `[app]` being used?

Also, the conditional is wrong, right now it means: Has bot suffix or (has app suffix and user == trustedApp). You need to put the first two checks in braces to achieve (has bot suffix or has app suffix) and user == trustedApp)

Reply:
No, it was an assumption based on the previous conversation since all GitHub Apps I've experienced only had `[bot]` suffix. Couldn't find any docs about possible suffixes. I have removed this check. Thanks for pointing that out!